skimage/feature/register_translation.py

```diff
@@ -196,7 +196,6 @@ def register_translation(src_image, target_image, upsample_factor=1,
     # If upsampling > 1, then refine estimate with matrix multiply DFT
     else:
         # Initial shift estimate in upsampled grid
-        shifts = np.round(shifts * upsample_factor) / upsample_factor
```

Comment:
Shifts is computed from above from the following code:
```python
shifts = np.array(maxima, dtype=np.float64)	     shifts = np.array(maxima, dtype=np.float64)
shifts[shifts > midpoints] -= np.array(shape)[shifts > midpoints]
```
where maxima are guaranteed to be integers converted to floats. This line doesn't do anything in my mind.

Reply:
hmm, interesting. The function definitely returns something. I'll have to look into it further.